GitUI/Script/ScriptRunner.cs

```diff
@@ -411,6 +411,21 @@ private static bool RunScript(IWin32Window owner, GitModule module, ScriptInfo s
                             argument = argument.Replace(option, prompt.UserInput);
                         }
 
+                        break;
+                    case "{UserFiles}":
+                        using (FilePrompt prompt = new FilePrompt())
+                        {
+                            if (prompt.ShowDialog() == DialogResult.OK)
+                            {
+                                argument = argument.Replace(option, prompt.FileInput);
+                            }
+                            else
+                            {
+                                argument = argument.Replace(option, string.Empty);
+                                break;
```

Comment:
If the dialog is closed without any files supplied, the script should not run. I think here we need to `return false` here.
If that's the case, please invert the `if` to reduce nesting

Reply:
Done.